kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/ars_nouveau/glyph_recipe.js

```diff
@@ -1,28 +1,17 @@
 events.listen('recipes', (event) => {
     const recipes = [
-        {
-            tier: 'TWO',
-            input: 'minecraft:nether_brick',
-            output: 'ars_nouveau:glyph_dampen',
-            id: 'ars_nouveau:glyph_dampen'
-        }
     ];
 
     recipes.forEach((recipe) => {
-        recipe.id
-            ? event
-                  .custom({
-                      type: 'ars_nouveau:glyph_recipe',
-                      tier: recipe.tier,
-                      input: recipe.input,
-                      output: recipe.output
-                  })
-                  .id(recipe.id)
-            : event.custom({
-                  type: 'ars_nouveau:glyph_recipe',
-                  tier: recipe.tier,
-                  input: recipe.input,
-                  output: recipe.output
-              });
+        const re = 

```

Comment:
I can't accept this part :P It's shorter I know but `const re` pains me to read.

Reply:
I'll just change it myself actually.

Reply:
Really? That syntax is already used in quite a few places (e.g. in crushing.js) and it is considerably less clunky and easier to read than the original.

Well, your code base, your rules! 😅